app/src/main/java/it/ilker/repeatcard/ui/quiz/QuizFragment.kt

```diff
@@ -52,10 +51,12 @@ class QuizFragment : Fragment() {
         viewLifecycleOwner.lifecycleScope.launchWhenStarted {
             viewModel.state.collect { state ->
                 when (state) {
-                    is QuizState.Loading -> {}
+                    is QuizState.Loading -> showLoader()
                     is QuizState.Error -> showError()
                     is QuizState.Success -> {
-                        feedbackText.visibility = INVISIBLE
+                        progress_circular.visibility = View.VISIBLE
```

Comment:
Should be `GONE` I think.

Reply:
I have changed the same across all files , because import View and then again importing View.Visible etc , doesn't make any sense to me.

Reply:
I prefer having `INVISIBLE` instead of `View.INVISIBLE` for readability if you don't mind. Regardless, here the progress visibility should be `progress_circular.visibility = View.GONE` I think.

Reply:
Got it. Will do the same. all files will have the similar style , as before.